extensions/jdbc/src/main/java/io/deephaven/jdbc/JdbcTypeMapper.java

```diff
@@ -986,111 +927,108 @@ MappingCollection build() {
      */
     private static final Map<Integer, MappingCollection> dataTypeMappings;
     static {
-        final Map<Integer, MappingCollection> initMap = new HashMap<>(25);
-        initMap.put(Types.CHAR, MappingCollection.builder(String.class)
+
+        dataTypeMappings = Map.ofEntries(Map.entry(Types.CHAR, MappingCollection.builder(String.class)
```

Comment:
If you added a newline between `Map.ofEntries(` and `Map.entry(` and then auto-format, you would achieve pleasing symmetry with the rest of the block.